### PR TITLE
SPUI-1656 > Separator fix

### PR DESCRIPTION
**Pull request by gravio-inv**
UI - Правки по сепаратору на экране Sell - choose destination, в флоу Convert при выборе другого ассета

### CODE REVIEW

**Review by VovaMedvediev — COMMENTED** [review]
Translate PR name to English

**Comment by eli1stark** [comment]
And provide description in English as well

**Review by eli1stark — CHANGES_REQUESTED** [review]
The same issue as with SPUI-1648, please provide a descriptive name of this pr and in the future as well.

File: plugins/simple_kit/lib/src/asset_items/simple_asset_item.dart [hunk 1]
```diff
@@ -136,9 +136,14 @@ class SAssetItem extends StatelessWidget {
                   ),
                 )
               else
-                const SDivider(
-                  width: double.infinity,
-                ),
+                Padding(
+                  //padding: const EdgeInsets.symmetric(vertical: 10.0),
```
Comment (eli1stark) [inline plugins/simple_kit/lib/src/asset_items/simple_asset_item.dart:140]: remove comment

File: plugins/simple_kit/lib/src/asset_items/simple_asset_item.dart [hunk 2]
```diff
@@ -136,9 +136,14 @@ class SAssetItem extends StatelessWidget {
...
+                  padding: const EdgeInsets.only(top: 10.0, bottom: 10),
```
Comment (eli1stark) [inline plugins/simple_kit/lib/src/asset_items/simple_asset_item.dart:141]: provide trailing comma

**Review by eli1stark — CHANGES_REQUESTED** [review]

**Review by VovaMedvediev — APPROVED** [review]

**Pull request merged**